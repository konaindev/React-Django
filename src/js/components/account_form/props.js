export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
