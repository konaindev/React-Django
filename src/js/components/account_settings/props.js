const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
