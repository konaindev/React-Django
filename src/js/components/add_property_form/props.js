export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
