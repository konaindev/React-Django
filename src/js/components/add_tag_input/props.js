const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
