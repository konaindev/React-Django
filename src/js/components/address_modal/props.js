export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
