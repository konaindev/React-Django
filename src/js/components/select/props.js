const options = [
  { label: "Option 1", value: "option1" },
  { label: "Option 2", value: "option2" },
  { label: "Option 3", value: "option3" },
  { label: "Option 4", value: "option4" }
];

const styles = {
  container: provided => ({ ...provided, width: 400 })
};

export const props = {
  options,
  styles,
  placeholder: "Select an option...",
  name: "select"
};
