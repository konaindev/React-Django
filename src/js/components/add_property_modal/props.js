import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
