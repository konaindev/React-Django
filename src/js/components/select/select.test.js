import React from "react";
import renderer from "react-test-renderer";

import Select, { SelectSearch } from "./index";
import { props } from "./props";

describe("Select", () => {
  it("render select", () => {
    const tree = renderer.create(<Select {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("render highlight select", () => {
    const tree = renderer
      .create(<Select theme="highlight" {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("render select search", () => {
    const tree = renderer.create(<SelectSearch {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("render highlight select search", () => {
    const tree = renderer
      .create(<SelectSearch theme="highlight" {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
