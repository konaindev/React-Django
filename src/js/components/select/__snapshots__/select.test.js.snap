// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select render gray select 1`] = `
<div
  className="select select--gray css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <input
        className="css-62g3xt-dummyInput"
        disabled={false}
        id="react-select-4-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        tabIndex="0"
        value=""
      />
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;

exports[`Select render highlight select 1`] = `
<div
  className="select select--highlight css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <input
        className="css-62g3xt-dummyInput"
        disabled={false}
        id="react-select-3-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        tabIndex="0"
        value=""
      />
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;

exports[`Select render highlight select search 1`] = `
<div
  className="select-search select select--highlight css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <div
        className="css-zz0hea-Input"
      >
        <div
          className="select__input"
          style={
            Object {
              "display": "inline-block",
            }
          }
        >
          <input
            aria-autocomplete="list"
            autoCapitalize="none"
            autoComplete="off"
            autoCorrect="off"
            disabled={false}
            id="react-select-8-input"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            spellCheck="false"
            style={
              Object {
                "background": 0,
                "border": 0,
                "boxSizing": "content-box",
                "color": "inherit",
                "fontSize": "inherit",
                "label": "input",
                "opacity": 1,
                "outline": 0,
                "padding": 0,
                "width": "1px",
              }
            }
            tabIndex="0"
            type="text"
            value=""
          />
          <div
            style={
              Object {
                "height": 0,
                "left": 0,
                "overflow": "scroll",
                "position": "absolute",
                "top": 0,
                "visibility": "hidden",
                "whiteSpace": "pre",
              }
            }
          >
            
          </div>
        </div>
      </div>
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;

exports[`Select render multi value select 1`] = `
<div
  className="select select--is-multi select--transparent css-2b097c-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container select__value-container--is-multi select__value-container--has-value css-1ux8av0"
    >
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 1
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 2
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 3
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 4
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 5
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 6
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 7
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 8
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 9
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <div
        className="select-multi-value"
      >
        <div
          className="select-multi-value__label"
        >
          Option 10
        </div>
        <div
          className="select-multi-value__close"
          onClick={[Function]}
          onMouseDown={[Function]}
          onTouchEnd={[Function]}
        >
          <svg
            aria-hidden="true"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 13 14"
            width="1em"
          >
            <path
              d="M1.43532 1.92201L11.3649 12.0613"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
            <path
              d="M11.4774 1.90854L1.83659 12.0504"
              stroke="currentColor"
              strokeLinecap="round"
              strokeWidth="2"
            />
          </svg>
        </div>
      </div>
      <input
        className="css-62g3xt-dummyInput"
        disabled={false}
        id="react-select-6-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        tabIndex="0"
        value=""
      />
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <div
        aria-hidden="true"
        className="select__indicator select__clear-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="css-tj5bde-Svg"
          focusable="false"
          height={20}
          viewBox="0 0 20 20"
          width={20}
        >
          <path
            d="M14.348 14.849c-0.469 0.469-1.229 0.469-1.697 0l-2.651-3.030-2.651 3.029c-0.469 0.469-1.229 0.469-1.697 0-0.469-0.469-0.469-1.229 0-1.697l2.758-3.15-2.759-3.152c-0.469-0.469-0.469-1.228 0-1.697s1.228-0.469 1.697 0l2.652 3.031 2.651-3.031c0.469-0.469 1.228-0.469 1.697 0s0.469 1.229 0 1.697l-2.758 3.152 2.758 3.15c0.469 0.469 0.469 1.229 0 1.698z"
          />
        </svg>
      </div>
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;

exports[`Select render select 1`] = `
<div
  className="select css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <input
        className="css-62g3xt-dummyInput"
        disabled={false}
        id="react-select-2-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        tabIndex="0"
        value=""
      />
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;

exports[`Select render select search 1`] = `
<div
  className="select-search select css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <div
        className="css-zz0hea-Input"
      >
        <div
          className="select__input"
          style={
            Object {
              "display": "inline-block",
            }
          }
        >
          <input
            aria-autocomplete="list"
            autoCapitalize="none"
            autoComplete="off"
            autoCorrect="off"
            disabled={false}
            id="react-select-7-input"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            spellCheck="false"
            style={
              Object {
                "background": 0,
                "border": 0,
                "boxSizing": "content-box",
                "color": "inherit",
                "fontSize": "inherit",
                "label": "input",
                "opacity": 1,
                "outline": 0,
                "padding": 0,
                "width": "1px",
              }
            }
            tabIndex="0"
            type="text"
            value=""
          />
          <div
            style={
              Object {
                "height": 0,
                "left": 0,
                "overflow": "scroll",
                "position": "absolute",
                "top": 0,
                "visibility": "hidden",
                "whiteSpace": "pre",
              }
            }
          >
            
          </div>
        </div>
      </div>
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;

exports[`Select render transparent select 1`] = `
<div
  className="select select--transparent css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <input
        className="css-62g3xt-dummyInput"
        disabled={false}
        id="react-select-5-input"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        tabIndex="0"
        value=""
      />
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;

exports[`Select render transparent select search 1`] = `
<div
  className="select-search select select--transparent css-1ym6uuo-container"
  onKeyDown={[Function]}
>
  <div
    className="select__control css-yk16xz-control"
    onMouseDown={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="select__value-container css-1jn1ih4"
    >
      <div
        className="select__placeholder css-1wa3eu0-placeholder"
      >
        Select an option...
      </div>
      <div
        className="css-zz0hea-Input"
      >
        <div
          className="select__input"
          style={
            Object {
              "display": "inline-block",
            }
          }
        >
          <input
            aria-autocomplete="list"
            autoCapitalize="none"
            autoComplete="off"
            autoCorrect="off"
            disabled={false}
            id="react-select-9-input"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            spellCheck="false"
            style={
              Object {
                "background": 0,
                "border": 0,
                "boxSizing": "content-box",
                "color": "inherit",
                "fontSize": "inherit",
                "label": "input",
                "opacity": 1,
                "outline": 0,
                "padding": 0,
                "width": "1px",
              }
            }
            tabIndex="0"
            type="text"
            value=""
          />
          <div
            style={
              Object {
                "height": 0,
                "left": 0,
                "overflow": "scroll",
                "position": "absolute",
                "top": 0,
                "visibility": "hidden",
                "whiteSpace": "pre",
              }
            }
          >
            
          </div>
        </div>
      </div>
    </div>
    <div
      className="select__indicators css-1g48xl4-IndicatorsContainer"
    >
      <span
        className="select__indicator-separator css-1okebmr-indicatorSeparator"
      />
      <div
        aria-hidden="true"
        className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
        onMouseDown={[Function]}
        onTouchEnd={[Function]}
      >
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--down"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 1.16134L4.92334 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 1.16134L5 4.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="select__dropdown-arrow select__dropdown-arrow--up"
          color="inherit"
          focusable="false"
          height="1em"
          role="presentation"
          viewBox="0 0 10 5"
          width="1em"
        >
          <path
            d="M1 3.83866L4.92334 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
          <path
            d="M8.82812 3.83866L5 0.5"
            stroke="currentColor"
            strokeLinecap="round"
          />
        </svg>
      </div>
    </div>
  </div>
  <input
    name="select"
    type="hidden"
    value=""
  />
</div>
`;
